Sofa/framework/Geometry/src/sofa/geometry/Triangle.h

```diff
@@ -71,6 +71,45 @@ struct Triangle
         }
     }
 
+
+    /**
+    * @brief	Compute the barycentric coefficients of the input point according to the Triangle vertices: (n0, n1, n2)
+    * @tparam   Node iterable container
+    * @tparam   T scalar
+    * @param	p0: position of the input point to compute the coefficients
+    * @param	n0, n1, n2: nodes of the triangle
+    * @return	sofa::type::Vec<3, T> barycentric coefficients 
+    */
+    template<typename Node,
+        typename T = std::decay_t<decltype(*std::begin(std::declval<Node>()))>,
+        typename = std::enable_if_t<std::is_scalar_v<T>>
+    >
+        static constexpr auto pointBaryCoefs(const Node& p0, const Node& n0, const Node& n1, const Node& n2)
```

Comment:
`pointBaryCoefs`, the name looks... not really informative and uses abbreviations.
Maybe something like `getBarycentricCoefficients()` or even `getBarycentricCoordinates()` which seems more in par with the maths

Reply:
Done and add more description on the method (based from wikipedia)